src/masternode-payments.h

```diff
@@ -24,7 +24,7 @@ static const int MNPAYMENTS_SIGNATURES_TOTAL            = 10;
 // V1 - Last protocol version before update
 // V2 - Newest protocol version
 static const int MIN_MASTERNODE_PAYMENT_PROTO_VERSION_1 = 70206;
```

Comment:
shouldn't this also be bumped to 70208?

Reply:
Yes, that's how we did this in the past and why it says `V1 - Last protocol version before update` BUT I was thinking about it a bit more and it seems to me that we probably should NOT bump this in the time of major migration to have the same _base_ with old nodes when spork10 is off and instead we should bump it in a minor pre-major update (if at all, maybe we could just leave it static forever?).